dwave/gate/mixedproperty.py

```diff
@@ -12,6 +12,11 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
+""":class:`mixedproperty` decorator.
```

Comment:
I'm not sure we should promote this tiny utility function in the public docs. Alternatively, move the module to something like utils submod?

I like to treat **public docs as API boundary** -- what's not documented publicly it's considered internal. We are OK to break it without triggering a major release.

We never officially ratified this as our public policy, but we've been following it for the most part in Ocean.

Reply:
I agree with this sentiment. I'm actually not that fond of utils modules/files, and would actually prefer not to have a `utils.py` at all. Currently, it only contains a temporary patch for a Python-version restricted `cached_property` decorator.

I also agree with `mixedproperty` not necessarily having to be promoted as a public class, although I think I'm actually fine with that. Perhaps we can just have Sphinx ignore it and simply not render it instead?

Reply:
Sure, whatever works for you, I don't have a strong opinion on how you implement it.